app/models/event.rb

```diff
@@ -27,12 +27,31 @@ class << self
     private :create
 
     def rehydrate!(params)
+      check_user(params) unless HostEnv.production?
+
       new(params).save!
     end
 
     def latest_decision
       order(created_at: :desc).find_by(event_type: 'Event::Decision')
     end
+
+    private
+
+    def check_user(params)
```

Comment:
Given this method is doing something a bit yick (in that we really wouldn't want this happening in production), can we flag it up with a less innocuous method name? `create_dummy_user_if_user_id_not_recognised` is maybe going a bit far, but you get the picture - something to make clear that this is a slightly risky procedure.